BUILD.bazel

```diff
@@ -167,6 +167,7 @@ copy_to(
         "//cmd/gapit",
         "//tools/build:build.properties",
         "//cmd/device-info:device-info",
+        "//vulkan_generator",
```

Comment:
I don't think this belongs here. We will not be distributing the code generator in the final package.